python/lsst/pipe/tasks/configurableActions/_configurableActionField.py

```diff
@@ -63,6 +63,13 @@ def __set__(self, instance, value, at=None, label="assignment"):
         history = instance._history.setdefault(self.name, [])
         history.append(("config value set", at, label))
 
+    def save(self, outfile, instance):
+        # docstring inherited from parent
+        value = self.__get__(instance)
+        fullname = _joinNamePath(instance._name, self.name)
+        outfile.write(f"{fullname}={_typeStr(value)}\n")
+        super().save(outfile, instance)
```

Comment:
It looks like this implementation is already doing everything it needs to do, and delegating it to `super` will just do some of those things again (incorrectly, I assume).  Can you add a comment explaining why you're still delegating to `super` here, and why that doesn't cause the field state to be written again?

Reply:
> No where does the Config type itself get assigned in the persisted config, leading to it always defaulting.

In the base `ConfigField`, the type of the config is completely static, so I don't think this is necessarily a problem there, though it obviously is for `ConfigurableAction`.